lib/cmd/common/apps.js

```diff
@@ -67,17 +73,12 @@ function oldApps(args, cb){
   }
 
   var action = args[0];
-  if (action === 'read') {
-    return cb("The 'fhc apps read' command is deprecated. Please use 'fhc read' command instead.");
-  }else if (action === 'create'){
-    return cb("The 'fhc apps create' command is deprecated. Please use 'fhc create' command instead.");
-  }else if (action === 'update'){
-    return cb("The 'fhc apps update' command is deprecated. Please use 'fhc update' command instead.");
-  }else if (action === 'delete'){
-    return cb("The 'fhc apps delete' command is deprecated. Please use 'fhc delete' command instead.");
-  } else if (action === 'ping'){
-    return cb("The 'fhc apps ping' command is deprecated. Please use 'fhc ping' command instead.");
-  } else{
+  if (action === 'read') return cb(errorMessageString(action));
+  else if (action === 'create') return cb(errorMessageString(action));
+  else if (action === 'update') return cb(errorMessageString(action));
+  else if (action === 'delete') return cb(errorMessageString(action));
+  else if (action === 'ping') return cb(errorMessageString(action));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either merge this branch with the identical one on line "78" or change one of the implementations. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1871)